stock_auto_move/tests/test_stock_auto_move.py

```diff
@@ -69,8 +69,11 @@ def test_10_auto_move(self):
             }
         )
         move3._action_confirm()
+        self.env["stock.quant"]._update_available_quantity(
```

Comment:
@ferran-S73 Why is this necessary ?

Reply:
@rousseldenis The move was not reserving stock because there was no available stock of the product. Maybe in older versions some other module added stock for the product?